testsuite/features/step_definitions/command_steps.rb

```diff
@@ -453,7 +457,11 @@
       sleep checking_rate
     end
   rescue StandardError => e
-    log e.message # It might be that the MU repository is wrong, but we want to continue in any case
+    log e.message
+    # It might be that the MU repository is wrong, but we want to continue in any case
```

Comment:
same, this remark applies only in the BV case

Reply:
Done